resources/public/api/conf/3.0/schemas/amendLossClaimsOrder/amend_response.json

```
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Amend Loss Claims Order Response",
+  "description": "Amend Loss Claims Order Response",
+  "type": "object",
+  "properties": {
+    "links": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A list of endpoint links that indicate possible actions related to the current resource.",
+        "properties": {
+          "href": {
+            "description": "The relative url of the endpoint",
+            "type": "string",
+            "example": "/individuals/losses/{nino}/loss-claims/order/{taxYearClaimedFor}"
```

Since we seem to have actual examples elsewhere...
```suggestion
            "example": "/individuals/losses/TC663795B/loss-claims/order/2022-23"
```